test/browser-transmit.test.js

```diff
@@ -50,7 +50,38 @@ test('passes send function the logged level', ({ end, is }) => {
 })
 
 test('passes send function messages in logEvent object', ({ end, same, is }) => {
-  const logger = pino({
+  // in order to cover all possible scenarios, need to test messages with/without objects, and with/without the `asObject` option set
```

Comment:
can you please break these into separate tests

Reply:
yes I will do that now